database/models/Priority.js

```diff
@@ -5,7 +5,7 @@ class Priority extends bookshelf.Model {
   get timestamps() { return true; }
 
   priorites() {
```

Comment:
This isn't necessary. This would make the give each priority and `.priorities` property.